_topic_map.yml

```diff
@@ -194,6 +194,8 @@ Topics:
     Distros: openshift-*
   - Name: Admission Controllers
     File: admission_controllers
+  - Name: Dynamic Admission Controllers
```

Comment:
Would it make sense to call them Admission Webhook instead? This use of "dynamic" is confusing. How does kube call them in the docs?

Reply:
yes, custom is better than "dynamic"